dataland-frontend/tests/e2e/support/TestUtility.ts

```diff
@@ -4,7 +4,7 @@ export interface ExecutionConfig {
   executionEnvironments: Array<ExecutionEnvironment>;
   dataEnvironments: Array<DataEnvironment>;
 }
-export type ExecutionEnvironment = "developmentLocal" | "development" | "preview";
+export type ExecutionEnvironment = "developmentLocal" | "ci" | "developmentCd" | "previewCd";
```

Comment:
Must be adapted after one of the next merges which incorporates the new dev environment. Also the instructions on the setup of a new server on the server wiki page need to be adapted for the new notation.

Reply:
Let's talk about that for a minute